### PR TITLE
Date picker v3

### PR DESCRIPTION
**Pull request by fragsalat**
- reimplemented date-picker as React/Preact component based on flatpickr
- Flatpickr only received modifications on the locales (Kw -> CW, Mon -> Mo, Tue -> Tu, ...)
- Style is based on latest design

20170315_date-picker_ v1_my

This is how it looks in the demo_app.js

date-picker-v2 1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,6 +91,9 @@
     "webpack-node-externals": "1.5.4",
     "wholesale-styleguide-scss": "git+ssh://git@github.com/wholesale-design-system/scss.git"
   },
+  "dependencies": {
+    "moment": "^2.0.0"
```
Comment (faboweb) [inline package.json]: you did remove momentjs as far as I understood. Is this then needed anymore?

Comment (fragsalat) [inline package.json]: Fixed. True

**Comment by fragsalat** [comment]
@faboweb What do you mean with min max borders? There is nothing implemented and also not required.

**Comment by faboweb** [comment]
@fragsalat With min/max-borders I mean defining a timeframe where you can pick a date from. Like: you can only select shipment between now and the end of the season. The older version had this implemented. I need this for the preorder recommendation frontend.

**Comment by fragsalat** [comment]
As far as I know this is not yet part of the styleguide. Can we merge this and place this as an enhancement?

**Pull request merged**